src/sqllogictest/src/main.rs

```diff
@@ -8,25 +8,50 @@
 // by the Apache License, Version 2.0.
 
 use std::cell::RefCell;
-use std::env;
 use std::fmt;
 use std::fs::File;
 use std::io::{self, Write};
+use std::path::PathBuf;
 use std::process;
 use std::sync::Arc;
 
 use chrono::Utc;
-use getopts::Options;
+use structopt::StructOpt;
 use tokio::runtime::Runtime;
 use walkdir::WalkDir;
 
 use sqllogictest::runner::{self, Outcomes, RunConfig, WriteFmt};
 use sqllogictest::util;
 
-const USAGE: &str = r#"usage: sqllogictest [PATH...]
-
-Runs one or more sqllogictest files. Directories will be searched
-recursively for sqllogictest files."#;
+#[derive(StructOpt)]
+/// Runs sqllogictest scripts to verify database engine correctness.
+struct Args {
```

Comment:
nit: I believe this is the only place you put the doc-comment under the `derive`.

Reply:
D'oh! Thanks!